src/NosCore.GameObject/Services/QuestService/QuestObjectives.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NosCore.Data.StaticEntities;
+using NosCore.Packets.Enumerations;
+using System;
+
+namespace NosCore.GameObject.Services.QuestService
+{
+    public class QuestObjectives : QuestObjectiveDto
+    {
+        public int? GetMaxCount(QuestType questType)
+        {
+            return questType switch
+            {
+                QuestType.Hunt or QuestType.SpecialCollect or QuestType.TimesSpace or QuestType.Product => SecondData != null && ThirdData != null ? (int?)Math.Floor((double)((SecondData + ThirdData) / 2)) : SecondData == null ? ThirdData : SecondData,
+                QuestType.Brings or QuestType.CaptureWithoutGettingTheMonster or QuestType.Capture  => ThirdData != null && FourthData != null ? (int?)Math.Floor((double)((ThirdData + FourthData) / 2)) : ThirdData == null ? FourthData : ThirdData,
```

Comment:
This looks super weird should be added to the parser instead. I still don’t get what maxCount is

Reply:
Need to rename first second third data if it’s holding something precise